.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,6 +3,12 @@
 
 <!-- (delete all below for minor changes) -->
 
+###### Type(s)
+<!-- update (title contains ": U(u)pdate to") and submission (new Portfile) are auto-detected, replace [ ] with [x] to select -->
+- [ ] bugfix
+- [ ] enhancement
+- [ ] security fix
```

Comment:
We can additionally offer auto-detection if the commit message matches `CVE-\d{4}-\d+`.